neqo-transport/src/connection/tests/stream.rs

```diff
@@ -4,9 +4,10 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use std::{cmp::max, collections::HashMap};
+use std::cmp::max;
 
 use neqo_common::{event::Provider as _, qdebug};
+use rustc_hash::FxHashMap as HashMap;
```

Comment:
What is the benefit of using `FxHashMap` in unit tests?

Reply:
None. I did a global replace :-) Will roll back for tests.